docs/DeveloperGuide.md

```diff
@@ -499,6 +499,194 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     
       Use case ends.
 
+**Use case: UC11 - Edit a travel plan**
+
+**MSS**
+
+1. User <ins>navigates to a specific travel plan (UC1)</ins> 
+2. User requests to edit travel plan
+3. Wanderlust shows that the travel plan has been edited
+
+   Use case ends.
+    
+**Extensions**
+
+*  1a. User is at the top directory.
```

Comment:
I'm a bit not sure about this. Is it we can edit the travel plan at top directory also?

Reply:
I was thinking that since we can view the travel plans at the top directory, we can also modify the start and end date there too. Or is it better to restrict the editing to within travel plan directory?

Reply:
dk hahaha

Reply:
@jiaweiteo @jeannetoh99 @timjkong

Reply:
i think this is ok but we should have a command to edit start and end date within travel plan directory also if not it might be a bit confusing